### DISCUSSION TITLE
Is possible integrate a multidatabase support to Model serializer ?

### DISCUSSION THREAD

**Discussion by gjae** [encode/django-rest-framework#9136] [Ideas & Suggestions]
Think that: add a __using_database__ propierty with "default" as default value, this could be util for integrate a multi database support easier.

For exameple; in the line `instance = ModelClass._default_manager.create(**validated_data)` could be ` instance = ModelClass._default_manager.using(self.using_database).create(**validated_data)`.

You think that's possible?

**Comment by IgorTeplov** [comment]
I think the get_db_name() method will be more flexible in this case.

def get_db_name(self):
     return "default"
